__tests__/app.test.js

```diff
@@ -245,4 +243,79 @@ describe("GET /api/reviews/:review_id/comments", () => {
             expect(response.body.msg).toBe("bad request");
         })
     });
-})
\ No newline at end of file
+});
+
+describe("POST /api/reviews/:review_id/comments", () => {
+    test('201: responds with the posted comment', () => {
+        const newComment = {
+            body: 'This is an example!',
+            username: 'mallionaire',
+        };
+        return request(app)
+        .post('/api/reviews/1/comments')
+        .send(newComment)
+        .expect(201)
+        .then((response) => {
+            const { comment } = response.body;
+            expect(comment).toEqual({
+                author: "mallionaire",
+                body: "This is an example!",
+                comment_id: 7,
+                created_at: expect.any(String),
+                review_id: 1,
+                votes: 0,
+            });
+        });
+    });
+    test('404: review_id in path does not exist', () => {
+        const newComment = {
+            body: 'This is an example!',
+            username: 'mallionaire',
+        };
+        return request(app)
+        .post("/api/reviews/99999/comments")
+        .send(newComment)
+        .expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe("not found");
+        });
+    });
+    test('404: username in path does not exist', () => {
+        const newComment = {
+            body: 'This is an example!',
+            username: 'JavierGarna',
+        };
+        return request(app)
+        .post("/api/reviews/1/comments")
+        .send(newComment)
+        .expect(404)
+        .then((response) => {
+            expect(response.body.msg).toBe("not found");
+        });
+    });
+    test('400: newComment does not contain both mandatory keys', () => {
+        const newComment = {
```

Comment:
You've only proved here that it errors if there is no username given, what happens if we dont give a body?